collect_app/src/androidTest/java/org/odk/collect/android/espressoutils/pages/FillBlankFormPage.java

```diff
@@ -49,4 +49,9 @@ public FillBlankFormPage checkIsFormSubtextDisplayed() {
         onView(withIndex(withId(R.id.form_subtitle2), 0)).check(matches(isDisplayed()));
         return this;
     }
+
+    public FormEntryPage clickOnForm(String formName) {
+        onView(withText(formName)).perform(click());
```

Comment:
This may require scrolling, I think, depending on what the form list looks like. If that sounds right, might as well address now.

Reply:
Yeah that's a good point. Will add that in.